docs/cicd/cicd-metrics.md

```diff
@@ -35,6 +36,25 @@ The conventions described in this section are specific to Continuous Integration
 **Disclaimer:** These are initial CICD metrics and attributes
 but more may be added in the future.
 
+### Guidance on per pipeline run metrics
```

Comment:
I have improved the guidance as discussed in the SemConv meeting 2025-08-04 (cf. [note](https://github.com/open-telemetry/semantic-conventions/pull/2237#issuecomment-3151921640) and [note](https://github.com/open-telemetry/semantic-conventions/pull/2237#discussion_r2251961406) in previous PR) by giving more examples of use cases and expanding on the description.

@lmolkova does it answer the question you had in the previous PR? https://github.com/open-telemetry/semantic-conventions/pull/2237#discussion_r2245903032